deployment/marketplace/digitialocean/one-click-droplet/Makefile

```diff
@@ -1,9 +1,12 @@
 RELEASE_NAME := vm-oneclick-droplet
 VM_VERSION ?= $(shell git describe --abbrev=0 --tags)
+PACKER_LOG := 1
+PACKER_LOG_PATH := packer.log
```

Comment:
Maybe we can use logging to `STDOUT` by default?
This will produce file which will be likely accidentally committed some day.

Reply:
Ok, why not =)